cmd/compose/convert.go

```diff
@@ -90,7 +91,8 @@ func convertCommand(p *projectOptions, backend api.Service) *cobra.Command {
 	}
 	flags := cmd.Flags()
 	flags.StringVar(&opts.Format, "format", "yaml", "Format the output. Values: [yaml | json]")
-	flags.BoolVar(&opts.resolve, "resolve-image-digests", false, "Pin image tags to digests.")
+	flags.BoolVar(&opts.resolveImageDigests, "resolve-image-digests", false, "Pin image tags to digests.")
+	flags.BoolVar(&opts.resolvePaths, "resolve-paths", false, "Resolve relative paths")
```

Comment:
nit: I’m not sure based on the description it’s super obvious what it does 🤔 
(also descriptions seem to end with a `.`)

Reply:
I mean: why and when would a user want to enable this?

Reply:
any better suggestion for flag description?

Reply:
Maybe we don’t really need this as a flag, actually?

Reply:
Sure, this is not actually a requirement to expose this to the end-user.
I just used this in the past to help diagnose relative path issues, but that's only me :P